docs/modules/ROOT/pages/how-tos/workflows/git-submodules.adoc

```diff
@@ -1,34 +1,72 @@
 = Building upstream projects with git submodules
 
-This document outlines some basic guidelines for synchronizing changes between an upstream and downstream repository using Git submodules.
+This document outlines a basic guide for synchronizing changes between an upstream and downstream repository using Git submodules.
 
 By leveraging Git submodules, it is possible to maintain a consistent and up-to-date relationship between related repositories (upstream and downstream) while having more control of how the project is built, tested and released.
 
 Additionally, {ProductName} can automate key aspects of this process, ensuring that changes from the upstream repository are efficiently and reliably propagated to the downstream repository.
 
-== Implementing this guideline
-=== Basic definitions
+== Basic definitions
 
 - **Upstream repository**: a community project which you do not have push access to (or which you do not want to push CI changes into).
 
-- **Downstream repository**: a self-owned copy of the upstream repository whose build, test, release lifecycle you want to control completely.
+- **Downstream repository**: a self-owned copy of the upstream repository whose build, test, release lifecycle you control completely.
 
-=== Create a downstream repository
+== Benefits of this guide
 
-The downstream repository can be used to do the following:
+1. **Separation of Concerns**: The downstream repository can stay focused on its specific development goals while the submodule handles its own code. This keeps your downstream repository clean and modular.
+2. **Automatic Dependency Syncing with link:https://github.com/renovatebot/renovate[Renovate]**: By automating the dependency sync process, whether its syncing the git submodule, the Containerfile, etc. your downstream repository can always use the latest code from the upstream repository(s), ensuring that your project benefits from new features, bug fixes, and security updates.
+3. **Version Control for upstream repositories**: You can lock the submodule to a specific commit or version, giving you control over when and how updates are integrated. This is especially useful if an upstream update introduces breaking changes.
+4. **{ProductName} CI**: Automated building and testing of your software artifacts help ensure that changes in the upstream repository don't break your downstream project.
+
+== Potential drawbacks
+
+1. **Complexity**: Managing submodules and automating their updates adds complexity to your workflow, such as:
+  - Debugging issues related to submodule updates
+  - Your automation configuration can be more challenging.
+  - Updates in the submodules can be opaque which can result in drift between your build process and that of the original repository.
```

Comment:
I made a comment about a potential solution for this here: https://github.com/konflux-ci/olm-operator-konflux-sample/blob/main/docs/konflux-onboarding.md#enable-drift-detection-optional

Reply:
Let's link there!